libs/blocks/chart/chart.js

```diff
@@ -154,10 +165,18 @@ const init = async (el) => {
   if (chartType !== 'oversizedNumber') {
     loadScript('/libs/deps/echarts.min.js')
       .then(() => {
+        const options = {
```

Comment:
It might be nice to make this more specific like `observerOptions` considering we also have chart options